src/components/TopBar/TopBarMediaControls.vue

```diff
@@ -272,7 +277,7 @@ export default {
 				return t('spreed', 'No screensharing')
 			}
 
-			return this.model.attributes.localScreen
+			return this.isScreensharing
```

Comment:
There's a couple more usages in this file, but I'd agree with you to not change them - they look consistent for now